languages/ru/tl_nc_language.php

```diff
@@ -25,6 +25,8 @@
 $GLOBALS['TL_LANG']['tl_nc_language']['edit']['1']                 = 'Редактировать язык ID %s.';
 $GLOBALS['TL_LANG']['tl_nc_language']['email_html']['0']           = 'HTML';
 $GLOBALS['TL_LANG']['tl_nc_language']['email_html']['1']           = 'Введите HTML.';
+$GLOBALS['TL_LANG']['tl_nc_language']['email_external_images']['0']= 'Внешние изображения';
+$GLOBALS['TL_LANG']['tl_nc_language']['email_external_images']['1']= 'Не вставлять изображения в HTML письма.';
```

Comment:
Please do not add any translations except for English. Translations are managed using transifex :)